app/pods/components/documents/report-link/template.hbs

```diff
@@ -162,6 +162,7 @@
               as |piece|
               }}
                 <WebComponents::VlDocument
+                  @onDeletePieceFromContainer={{action this.setPreviousPiece}}
```

Comment:
de `action` helper zou binnenkort deprecated worden.. best vervangen door een equivalent https://guides.emberjs.com/release/upgrading/current-edition/action-on-and-fn/

Reply:
Ik zou dit als een globale change zien in de stack. Momenteel geen showstopper denk ik.